.github/create-releases.js

```diff
@@ -26,14 +28,6 @@ const repoMeta = {
     repo: "microsoft-authentication-library-for-js"
 };
 
-const libFolders = [
```

Comment:
Any objections to supporting both this hard coded list and providing the name as an environment variable? It would make it easier if this script needs to be run manually locally, i.e. use the environment variable if provided, fallback to the existing hard-coded list.

Reply:
Thanks!